apps/outlook/src/main.tsx

```diff
@@ -0,0 +1,38 @@
+import "./globals.css";
+
+import {
+  createMemoryHistory,
+  Link,
+  Outlet,
+  ReactLocation,
+  Router,
+} from "@tanstack/react-location";
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { TRPCProvider } from "./features/trpc/provider";
+
+import { getRoutes } from "./features/router/config";
+
+const location = new ReactLocation({
+  history: createMemoryHistory({
```

Comment:
For outlook we can even use a regular URL router I suppose, right?

Not that it matters much as the user will not access URLs explicitly (we might want to in dev). I also see a plus point of keeping consistency with the `extension` and therefore having both be in-memory routers